### PR TITLE
Onetag bid Adapter: added video support.

### PR DESCRIPTION
**Pull request by OneTagDevOps**
## Type of change

- [ ] Bugfix
- [X] Feature
- [ ] New bidder adapter   
- [X] Code style update (formatting, local variables)
- [X] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Added video support for OnetagBidAdapter. Minor code refactoring.

- contact email of the adapter’s maintainer devops@onetag.com

### CODE REVIEW

**Comment by OneTagDevOps** [comment]
PR docs: https://github.com/prebid/prebid.github.io/pull/1863

File: modules/onetagBidAdapter.js [hunk 1]
```diff
@@ -6,13 +6,13 @@ const ENDPOINT = 'https://onetag-sys.com/prebid-request';
 const USER_SYNC_ENDPOINT = 'https://onetag-sys.com/usync/';
 const BIDDER_CODE = 'onetag';
 const BANNER = 'banner';
+const VIDEO = 'video';
 
 // =======
```
Comment (Fawke) [inline modules/onetagBidAdapter.js]: Are these comment lines necessary?

Comment (OneTagDevOps) [reply modules/onetagBidAdapter.js]: Hello @Fawke , we've just updated the pull request following your suggestions and added the test parameters for the video case

**Comment by OneTagDevOps** [comment]
Hello @Fawke , we've just updated the pull request following your suggestions and added the test parameters for the video case

File: modules/onetagBidAdapter.js [hunk 2]
```diff
@@ -6,13 +6,7 @@ const ENDPOINT = 'https://onetag-sys.com/prebid-request';
 const USER_SYNC_ENDPOINT = 'https://onetag-sys.com/usync/';
 const BIDDER_CODE = 'onetag';
 const BANNER = 'banner';
```
Comment (Fawke) [inline modules/onetagBidAdapter.js]: You can import the mediaType (both, BANNER and VIDEO) from `src/mediaTypes.js` file. Please check the appnexusBidAdapter.js for more details.

Comment (Fawke) [reply modules/onetagBidAdapter.js]: @OneTagDevOps

Comment (OneTagDevOps) [reply modules/onetagBidAdapter.js]: @Fawke just updated

**Comment by OneTagDevOps** [comment]
@Fawke now you should see the test video response, please check it

File: modules/onetagBidAdapter.js (hunk 2 above)
Comment (Fawke) [reply modules/onetagBidAdapter.js]: Thanks!

**Comment by Fawke** [comment]
@OneTagDevOps 
Still seeing a Banner response to a instream request. If it helps, am sharing the instream test file that am using to check your response!

Screenshot 2020-03-24 at 1 46 49 PM

[Link](https://gist.github.com/Fawke/aaa5ac6bce8fd05d724c8d4ee333899b) to test page

**Comment by OneTagDevOps** [comment]
Hello @Fawke , thank you! We checked and it looks like you were getting an old response. It should be updated now. Can you please check again?

**Pull request merged**